multimedia/mlt/Makefile

```diff
@@ -11,105 +11,85 @@ COMMENT?=	Multimedia framework for TV broadcasting
 LICENSE?=	GPLv2 GPLv3 LGPL21
 LICENSE_COMB?=	multi
 
-USES+=		alias compiler:c++11-lang eigen:3 gmake gnome localbase \
-		pkgconfig
-GNU_CONFIGURE=	yes
-CONFIGURE_ARGS+=--enable-gpl --enable-gpl3 --target-os=FreeBSD
+USES+=		alias cmake compiler:c++11-lang eigen:3 gnome pkgconfig
 
-.ifndef(WITH_DEBUG)
-CONFIGURE_ARGS+=--disable-debug
-.else
-CONFIGURE_ARGS+=--enable-debug
-.endif
-
-.ifndef(SLAVEPORT) # Master port.
+# It's still mlt, but installs support files to mlt-7
+DATADIR=	${PREFIX}/share/${PORTNAME}-7
 
 # They both install bin/melt.
-CONFLICTS_INSTALL=	freeze-[0-9]*
+CONFLICTS_INSTALL=	freeze-[0-9]* \
+			mlt-6*
```

Comment:
I don't think the conflict should be necessary -- we should make this coninstallable.